packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,97 @@
+:host {
```

Comment:
**issue**: ik mis de height op basis van de `--way-form-element-height-medium` (en `--way-form-element-height-small` bij de small size en `--way-form-element-height-large` bij de large size. Dit zorgt er voor dat de buttons dezelfde hoogte hebben als de button etc. Zorgt voor consistentie in het design. Zie